docs/data_management/res/cheaha_individual_account.csv

```diff
@@ -0,0 +1,6 @@
+Permissions/Responsibilities,Research Computing,User, Storage Owner's Supervisor
```

Comment:
As discussed, let's flip the table upside-down so the most-full rows are at the top, least full at the bottom. Move RC responsibilities to the right-most column.